monitorTypeSurfaces_Explainer.md

```diff
@@ -0,0 +1,23 @@
+## Summary
+When `getDisplayMedia()` is called, the browser offers the user a choice of display surfaces: tabs, windows, or monitors. Using the `monitorTypeSurfaces` option, the web application may now hint to the browser if it prefers to include display surfaces whose type is monitor among the choices offered to the user. The browser may still ignore this hint. Web applications are therefore encouraged to still check the displaySurface setting of the tracks they receive.
+
+* The relevant spec-change is [here](https://github.com/w3c/mediacapture-screen-share/pull/274/files).
+* The resulting text in the spec is [here](https://w3c.github.io/mediacapture-screen-share/#dom-displaymediastreamoptions-monitortypesurfaces).
+
+## Motivation
+* Protect the user from the pitfall of choosing a source which the app will reject eventually.
+* Protect companies from leakage of private information through employee-error.
+
+## Activation
+```js
+// Hint the browser to exclude monitors from the choices offered to the user.
+const stream = await navigator.mediaDevices.getDisplayMedia({
+  monitorTypeSurfaces: "exclude",
+});
+
+// We still need to check though...
+const { displaySurface } = stream.getTracks()[0].getSettings();
+if (displaySurface == "monitor") {
+  // User has picked a monitor...
+}
+```
```

Comment:
Wdyt of adding a "Result (Chrome)" section and image, as in [this other explainer](https://github.com/eladalon1983/screen-share-explainers/blob/main/systemAudio_Explainer.md#result-chrome)?

Reply:
I'm not sure how GitHub will handle URLs uploaded from my repo. Shall I share offline images you can upload later once the PR is merged?

```
## Result (Chrome)
![image1](https://github.com/eladalon1983/screen-share-explainers/assets/22117736/c4c3835a-254b-44d8-b9ca-c719a95ec1dd)
![image2](https://github.com/eladalon1983/screen-share-explainers/assets/22117736/4e7c4580-3113-4f53-93b5-955daa293ad9)
```

Reply:
Works for me.